services/mailgun.js

```diff
@@ -1,8 +1,16 @@
-// import Mailgun from 'mailgun-js';
-// import configs from 'configs/index';
+import Mailgun from 'mailgun-js';
+import configs from 'configs/index';
 
-// const { MAILGUN_API_KEY, DOMAIN } = configs;
+const { MAILGUN_API_KEY, DOMAIN } = configs;
 
-// const mailgun = Mailgun({ apiKey: MAILGUN_API_KEY, domain: DOMAIN });
+const mailgun = Mailgun({ apiKey: MAILGUN_API_KEY, domain: DOMAIN });
 
-// TODO: Setting up mailgun services here
+function send(data) {
+  const report = mailgun.messages().send(data);
+  return report;
+}
+
+export default {
+  default: mailgun,
+  send,
+};
```

Comment:
Not ok here.
Please create a class `MailgunService`, have `this.api` for mailgun and `this.send` for function send.
Then export an instance of `MailgunService`.

Reply:
Solved: https://github.com/EasyEventV2/ee2_api/pull/9/commits/409cebfa64b084cb7802c255ca941b8fb38eb2ed